Monika After Story/game/zz_reactions.rpy

```diff
@@ -2855,3 +2855,49 @@ label mas_reaction_gift_clothes_velius94_dress_whitenavyblue:
     if giftname is not None:
         $ store.mas_filereacts.delete_file(giftname)
     return
+
+init 5 python:
+    if not renpy.seen_label("mas_reaction_gift_carddeck"):
+        addReaction("mas_reaction_gift_carddeck", "carddeck", is_good=True)
```

Comment:
`carddeck` is generic, this can be any sort of deck. Like a traditional deck for playing games like go fish or war, etc.

We should make this more specific for if we add more card games in the future.

Reply:
we might just have to stick with `noudeck`, but considering how weird that seems as a gift, might be better to just not have this as a gift-unlocked game.

Reply:
I want a game that's being unlocked by the player, not Monika. From all the games I've planned, this makes the most sense to be a gift.